java/src/jmri/jmrit/beantable/BeanTableDataModel.java

```diff
@@ -37,20 +37,21 @@
 import javax.swing.JScrollPane;
 import javax.swing.JTable;
 import javax.swing.JTextField;
-import javax.swing.RowSorter;
+import javax.swing.RowSorter; // still used?
```

Comment:
This comment indicates that you didn't bother to review the code you are commenting on.